### ISSUE TITLE
Automate API package publication

### ISSUE THREAD

**Issue by arieljbirnbaum** [litentry/litentry-parachain#2090]
### Context

Part of #2087 
:warning: Blocked by #2089 

### Task

- Create a script (Makefile step? Yarn script? Shell script? TBD) that takes the currently built version of the API packages and publishes it to the npm registry. Refer to the outcome of #2089 for procedure details.
  - NOTE: the script should take a branch name as parameter, and use that to **tag** the published versions.
  - NOTE: publishing will fail if the version (as defined in `package.json`) already exists; do we need to handle this somehow?

### Acceptance criteria

Use the script to publish a new version of the packages.

**Comment by 0xverin** [comment]
>NOTE: publishing will fail if the version (as defined in package.json) already exists; do we need to handle this somehow?

Furthermore, I couldn't think of a good way to handle the failure cases, so I didn't make any updates. I think it's reasonable to submit a separate PR to modify the version if publish fails.

>the script should take a branch name as parameter, and use that to tag the published versions.

We can directly select the branch to execute GHA. So, we don't need parameters.

Q:We need three tags: `dev`, `staging`, `latest` (default). Should `latest` be based on which branch? Staging?


<img width="1342" alt="image" src="https://github.com/litentry/litentry-parachain/assets/104152026/78edb438-fa4b-4c40-98d0-e16380fb751f">
